docs/en/latest/deployments/kubesphere.md

```diff
@@ -50,6 +50,19 @@ helm install apisix apisix/apisix \
   --namespace ingress-apisix \
   --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
 kubectl get service --namespace ingress-apisix
+
+# Due to network reasons, you may have problems when deploying. You can consider using an offline method. Here is an example:
+
+wget https://github.com/apache/apisix-helm-chart/releases/download/apisix-1.3.0/apisix-1.3.0.tgz 
+tar xf apisix-1.3.0.tgz 
```

Comment:
```suggestion
VERSION=$(curl https://artifacthub.io/api/v1/packages/helm/apisix/apisix | jq -r ".version")
wget https://github.com/apache/apisix-helm-chart/releases/download/apisix-$VERSION/apisix-$VERSION.tgz 
tar xf apisix-$VERSION.tgz 
```

It might be better to use the latest version.